crates/topos/tests/snapshots/regtest__regtest_spam_help_display.snap

```diff
@@ -33,6 +33,12 @@ Options:
           Socket of the opentelemetry agent endpoint. If not provided open telemetry will not be used [env: TOPOS_OTLP_AGENT=]
       --otlp-service-name <OTLP_SERVICE_NAME>
           Otlp service name. If not provided open telemetry will not be used [env: TOPOS_OTLP_SERVICE_NAME=]
+      --benchmark
+          Flag to indicate usage of Kubernetes [env: TOPOS_NETWORK_SPAMMER_BENCHMARK=]
+      --dns <DNS>
+          DNS template for generating target node addresses. e.g. `--dns="http://validator-{N}:1340"` [env: TOPOS_NETWORK_SPAMMER_DNS=]
```

Comment:
Hmm, what about calling this `--hosts <HOST_TMPL>` instead? The fact that kubernetes wants host names and uses DNS to find peers is sort of a detail that isn't too relevant I think?